gui/projectfiledialog.cpp

```diff
@@ -808,7 +826,7 @@ void ProjectFileDialog::setLibraries(const QStringList &libraries)
 void ProjectFileDialog::addSingleSuppression(const SuppressionList::Suppression &suppression)
 {
     mSuppressions += suppression;
-    mUI->mListSuppressions->addItem(QString::fromStdString(suppression.getText()));
+    mUI->mListSuppressions->addItem(QString::fromStdString(suppressionAsText(suppression)));
```

Comment:
why don't we improve or use `Suppression::getText` ? I have the feeling that the serialization/deserialization is unit tested. suppressionAsText is not unit tested.

Reply:
It is a format that is only used by the GUI so it should not be in the common code. The "serialized" format is used in the IPC and that should be used consistently. I have not looked into it yet.